src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -1507,6 +1554,17 @@ int sd_dhcp_server_set_default_lease_time(sd_dhcp_server *server, uint32_t t) {
         return 1;
 }
 
+int sd_dhcp_server_set_ipv6_only_preferred_time(sd_dhcp_server *server, uint32_t t) {
+        assert_return(server, -EINVAL);
+        assert_return(t >= 300, -EINVAL);
+
+        if (t == server->ipv6_only_preferred_time)
+                return 0;
+
+        server->ipv6_only_preferred_time = t;
```

Comment:
I'd like to also rename `preferred_time` -> `preferred_sec` or so, to indicate the unit is seconds.